docs/.vuepress/config.js

```diff
@@ -80,21 +82,19 @@ module.exports = {
   // ssrTemplate: 'index.ssr.html',
   themeConfig: {
     logo: '/logo.png',
-    nextLinks: true,
-    prevLinks: true,
+    nextLinks: false,
+    prevLinks: false,
     repo: 'handsontable/hyperformula',
     docsRepo: 'handsontable/hyperformula',
     docsDir: 'docs',
-    docsBranch: 'develop',
+    docsBranch: 'master',
     editLinks: true,
     editLinkText: 'Help us improve this page',
     lastUpdated: false,
     smoothScroll: false,
     searchPlaceholder: 'Search...',
-    // algolia: {
-    //   apiKey: '<API_KEY>',
-    //   indexName: '<INDEX_NAME>'
-    // },
+    searchLimitApi: 5,
+    searchLimitGuide: 15,
```

Comment:
I think the API limit should be raised, and the Guide limit should be lowered.

There should be more API results when you add partial matching (poor man's "fuzzy search") as requested in my other comment. 

For example, there are many methods in the docs that match `cell`.

Reply:
I've set the limits to 10 for API and 10 for Guides